docs/host/host.md

```diff
@@ -21,6 +21,32 @@ Because Harvester is built on top of Kubernetes and uses etcd as its database, t
 
 ## Node Maintenance
 
+
+:::warning
+
+Due to a recent [bug](https://github.com/harvester/harvester/issues/7128), there is a possibility that virtual machines (VMs) on a node may encounter an I/O error while in maintenance mode. To mitigate this issue until a fix is released, we advise using the following maintenance procedure:
+
+1. Set the taint on the maintenance node:
+
+    ```sh
+    kubectl taint node <NODE> --overwrite kubevirt.io/drain=draining:NoSchedule
+    ```
+
+1. Wait for all VMs to be live-migrated out of the maintenance node.
```

Comment:
Shouldn't the order of these steps be reversed?

```
1. Set the taint on the maintenance node.
1. Select **Enable Maintenance Mode** in the UI.
1. Wait for all VMs to be live-migrated out of the maintenance node.
```

Reply:
@bk201 clarified this with me. All good. 

ref: 
- https://kubevirt.io/user-guide/cluster_admin/node_maintenance/#evict-all-vms-from-a-node
- https://github.com/harvester/harvester/pull/7133